docs: add build instructions

paulbutcher opened the pull request.

paulbutcher committed:
Add build instructions

magnus-madsen commented on README.md:
````diff
@@ -1,3 +1,26 @@
+npm ci 
+npm run build
+serve -s build
+```
````
Are these commands different from `npm start`?

magnus-madsen commented:
We could also add a comment to `src/Data.js` that this file should be obtained by running the compiler.

paulbutcher committed:
Address code review comments

paulbutcher commented:
I've added the comment to `src/Data.js` as suggested and changed the instructions to reference `npm start` (I was cribbing from the github workflow, and didn't work out that `npm start` would do the trick until afterwards).

magnus-madsen commented:
Thanks!
